common/src/state/mod.rs

```diff
@@ -452,13 +446,7 @@ impl State {
                 conversation_id,
                 mut message,
             } => {
-                if let Some(ids) = self
-                    .get_chat_by_id(conversation_id)
-                    .map(|c| self.chat_participants(&c))
-                {
-                    message.insert_did(&ids, &self.get_own_identity().did_key());
-                }
-                let ping = message.is_mention;
+                let ping = message.is_mention_self(&self.get_own_identity().did_key());
```

Comment:
Not related to this PR but might be best to move `self.get_own_identity()` outside of the match condition and borrow from that